src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Footer(props) {
```

Comment:
Align style, use either arrow function / traditional style in your code for writing components